internal/cmd/shell/shell.go

```diff
@@ -267,23 +266,12 @@ func formatMySQLBranch(database string, branch *ps.DatabaseBranch) string {
 }
 
 func historyFilePath(org, db, branch string) (string, error) {
-	dir, err := homedir.Dir()
+	historyFilePath := fmt.Sprintf(".pscale/history/%s.%s.%s", org, db, branch)
+
+	historyFile, err := xdg.DataFile(historyFilePath)
```

Comment:
should we fall back to using `homedir.Dir()` if we get an error from this, just to prevent that from stopping progress?

Reply:
Thinking I might have it check if the old directory is there first so we don't wipe peoples history when this ships.

Then if not, use the new method.

Reply:
I like that 👍